frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -441,6 +441,23 @@ const HostDetailsPage = ({
     ? teams?.find((t) => t.id === host.team_id)?.features
     : config?.features;
 
+  const getOSUpdatesFromMDMConfig = (hostPlatform: string) => {
```

Comment:
can we update this (and everything using `IHost.platform`) to be of type `Platform`?

```
export type Platform =
  | "darwin"
  | "windows"
  | "linux"
  | "chrome"
  | "ios"
  | "ipados";
  ```

Reply:
Right now IHost.platform everywhere is defined `platform: string;` so I followed that pattern

Confirmed it includes nuance Linux platforms like "rhel" and "ubuntu", but I didn't test if it includes all of the ones that can be returned from the API. I think I got time to prioritize fixing this type everywhere in our codebase.